Jenkinsfile

```diff
@@ -104,7 +104,7 @@ pipeline {
 						agent {
 							dockerfile {
 								dir "trustme/cml/scripts/ci"
-								args '--entrypoint=\'\' -v /yocto_mirror/sources-v0.9:/source_mirror -v /yocto_mirror/sstate-cache-v0.9:/sstate_mirror --env BUILDNODE="${env.NODE_NAME}"'
+								args '--entrypoint=\'\' -v /yocto_mirror/v0.9/x86/:/source_mirror -v /yocto_mirror/v0.9/x86/sstate-cache-v0.9:/sstate_mirror --env BUILDNODE="${env.NODE_NAME}"'
```

Comment:
Is the path to /source_mirror really correct? I can't see the 'sources' subdirectory that is present on the server.

_@_:/yocto_mirror/v0.9/x86$ ls -l
total 8
lrwxrwxrwx 1 _ _ 18 Apr  3 08:48 sources -> ../../sources-v0.9
lrwxrwxrwx 1 _ _ 23 Apr  3 08:49 sstate-cache -> ../../sstate-cache-v0.9